test/fixtures/hooks/test_hook.exs

```diff
@@ -0,0 +1,3 @@
+use Kitto.Hooks.DSL
+
+hook :hello, do: broadcast! :hello, %{text: "Hello World"}
```

Comment:
Tests print a warning about `variable conn is unused`. We have to do something about it.